bridge/contracts/Bridge/BridgeV1.sol

```diff
@@ -17,20 +17,20 @@ import "../zeppelin/token/ERC777/IERC777.sol";
 import "../zeppelin/utils/Address.sol";
 import "../zeppelin/math/SafeMath.sol";
 
-import "./IBridge_old.sol";
+import "./IBridgeV1.sol";
 import "../interface/ISideToken.sol";
 import "../interface/ISideTokenFactory.sol";
-import "./AllowTokens_old.sol";
-import "./Utils_old.sol";
+import "../AllowTokens/AllowTokensV1.sol";
+import "../Utils/UtilsV1.sol";
 
-contract Bridge_old is Initializable, IBridge_old, IERC777Recipient, UpgradablePausable, UpgradableOwnable, ReentrancyGuard {
+contract BridgeV1 is Initializable, IBridgeV1, IERC777Recipient, UpgradablePausable, UpgradableOwnable, ReentrancyGuard {
```

Comment:
This should be BridgeV2  https://github.com/rsksmart/tokenbridge/pull/263/files#diff-42699c4d471d9ab5f788196252f6ba98c1268374871e298566f1205f2ec1f655R73

Reply:
Should I add the Contract Bridge V1 as well? to the contract versions?

Reply:
Done at https://github.com/rsksmart/tokenbridge/pull/263/commits/0dcd4497fb42c40e3cde1a253333b7ad68cc14ec